PluralKit.Bot/CommandSystem/Context/Context.cs

```diff
@@ -121,14 +122,18 @@ public async Task Execute<T>(Command? commandDef, Func<T, Task> handler, bool de
         {
             await Reply($"{Emojis.Error} {e.Message}\n**Command usage:**\n> pk;{commandDef?.Usage}");
         }
+        catch (PKCancelError e)
+        {
+            await Reply($"{Emojis.Error} {e.Message}", null, null, true);
+        }
         catch (PKError e)
         {
             await Reply($"{Emojis.Error} {e.Message}");
         }
         catch (TimeoutException)
         {
             // Got a complaint the old error was a bit too patronizing. Hopefully this is better?
-            await Reply($"{Emojis.Error} Operation timed out, sorry. Try again, perhaps?");
+            await Reply($"{Emojis.Error} Operation timed out, sorry. Try again, perhaps?", null, null, true);
```

Comment:
i'd use `referenceCommand: true` just to avoid the `, null, null, ` - makes it clear which parameter is being set

Reply:
```suggestion
            await Reply($"{Emojis.Error} Operation timed out, sorry. Try again, perhaps?", referenceCommand true);
```